configurations/dev.ts

```diff
@@ -8,111 +8,118 @@ const checkOpenHours = true;
-    Read: "Visto",
-    MessageSendingDisabled: "El envío de mensajes ha sido desactivado",
-    Today: "HOY",
-    Yesterday: "AYER",
-    Save: "GUARDAR",
-    Reset: "RESETEAR",
-    MessageCharacterCountStatus: "{{currentCharCount}} / {{maxCharCount}}",
-    SendMessageTooltip: "Enviar Mensaje",
-    FieldValidationRequiredField: "Campo requerido",
-    FieldValidationInvalidEmail: "Por favor provea una dirección válida de email",
+  es: {
+    EntryPointTagline: 'Chatea con nosotros',
+    MessageCanvasTrayButton: 'EMPEZAR NUEVO CHAT',
+    InvalidPreEngagementMessage:
+      'Los formularios previos al compromiso no se han establecido y son necesarios para iniciar el chat web. Por favor configúrelos ahora en la configuración.',
+    InvalidPreEngagementButton: 'Aprende más',
+    InputPlaceHolder: 'Escribe un mensaje',
+    TypingIndicator: '{0} está escribiendo ... ',
+    Read: 'Visto',
+    MessageSendingDisabled: 'El envío de mensajes ha sido desactivado',
+    Today: 'HOY',
+    Yesterday: 'AYER',
+    Save: 'GUARDAR',
+    Reset: 'RESETEAR',
+    MessageCharacterCountStatus: '{{currentCharCount}} / {{maxCharCount}}',
+    SendMessageTooltip: 'Enviar Mensaje',
+    FieldValidationRequiredField: 'Campo requerido',
+    FieldValidationInvalidEmail: 'Por favor provea una dirección válida de email',
 
-    PreEngagementDescription: "Comencemos",
+    PreEngagementDescription: 'Comencemos',
 
-    WelcomeMessage: "¡Bienvenido a Aselo!",
-    MessageCanvasTrayContent:"",
+    WelcomeMessage: '¡Bienvenido a Aselo!',
+    MessageCanvasTrayContent: '',
     AutoFirstMessage: '',
   },
-  'dk': {
-    MessageCanvasTrayContent:"",
+  dk: {
+    MessageCanvasTrayContent: '',
     AutoFirstMessage: '',
   },
 };
 
 const preEngagementConfig: PreEngagementConfig = {
   description: "Let's get started",
-  fields:
-    [
-      {
-        label: "What is your helpline?",
-        type: "SelectItem",
-        attributes:
+  fields: [
+    {
+      label: 'What is your helpline?',
+      type: 'SelectItem',
+      attributes: {
+        name: 'helpline',
+        required: true,
+        readOnly: false,
+      },
+      options: [
+        {
+          value: 'Select helpline',
+          label: 'Select helpline',
+          selected: true,
+        },
         {
-          name: "helpline",
-          required: true,
-          readOnly: false
+          value: 'Fake Helpline',
+          label: 'Fake Helpline',
+          selected: false,
         },
-        options: [
-          {
-            value: "Select helpline",
-            label: "Select helpline",
-            selected: true
-          },
-          {
-            value: "Fake Helpline",
-            label: "Fake Helpline",
-            selected: false
-          },
-        ]
+      ],
+    },
+    {
+      label: 'What is your name?',
+      type: 'InputItem',
+      attributes: {
+        name: 'firstName',
+        required: true,
+        readOnly: false,
+        maxLength: 30,
```

Comment:
@janorivera Is this a good API to go with for limiting the accepted length for an input?

Reply:
@GPaoloni This is OK by me!, I think having the maxLength as part of the input attribute makes more sense to me. Thanks!